python/python/tests/test_async_client.py

```diff
@@ -5559,6 +5559,49 @@ async def test_getex(self, redis_client: TRedisClient):
         )
         assert await redis_client.ttl(key1) == -1
 
+    @pytest.mark.parametrize("cluster_mode", [True, False])
+    @pytest.mark.parametrize("protocol", [ProtocolVersion.RESP2, ProtocolVersion.RESP3])
+    async def test_lolwut(self, redis_client: TRedisClient):
+        result = await redis_client.lolwut()
+        assert "Redis ver. " in result
+        result = await redis_client.lolwut(parameters=[50, 20])
+        assert "Redis ver. " in result
+        result = await redis_client.lolwut(6)
+        assert "Redis ver. " in result
+        result = await redis_client.lolwut(5, [30, 4, 4])
+        assert "Redis ver. " in result
+
+        if isinstance(redis_client, RedisClusterClient):
```

Comment:
Maybe move this piece of test into another test case which runs on cluster client only.

Reply:
Both standalone and cluster tests are not huge (after adopting Aaron's suggestions). I kind of want to keep them in the same test.